pkg/snapshot/snapshotter/snapshotter.go

```diff
@@ -42,6 +43,15 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+const (
+	Min                                 = 0 // Minutes field
+	Hour                                = 1 // Hours field
+	DayOfMonth                          = 2 // Day of month field
+	Month                               = 3 // Month field
+	DayOfWeek                           = 4 // Day of week field
+	defaultFullSnapMaxTimeWindowInHours = 23.5
```

Comment:
If iota is used, declare this as separate constant. Can we rename this constant as DefaultFullSnapMaxTimeWindow ? Specifiy that the value is in hours inside a comment. No need to include "InHours" in variable name